packages/core/src/DirectoryLoader.ts

```diff
@@ -34,6 +30,16 @@ function toJSON(this: ICredentialType) {
 	};
 }
 
+type Codex = {
```

Comment:
What's a `Codex`? Could we add some description for it

Reply:
TBH, I still don't know for sure. These lines were moved from [here](https://github.com/n8n-io/n8n/blob/master/packages/core/src/DirectoryLoader.ts#L212-L220).